omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -236,6 +236,7 @@
 default['private_chef']['opscode-erchef']['keygen_timeout'] = 1000
 default['private_chef']['opscode-erchef']['keygen_key_size'] = 2048
 default['private_chef']['opscode-erchef']['strict_search_result_acls'] = false
+default['private_chef']['opscode-erchef']['ssl_session_caching']['enabled'] = true
```

Comment:
This feels a bit backwards.  This should be `false` and the conditional in the template should be `if !`